ccgains/trades.py

```diff
@@ -257,9 +267,43 @@ def _parse_trade(str_list, param_locs, default_timezone):
         else:
             pdict[key] = val
 
+    # Get missing fees if requested
+    if fill_missing_fees and pdict['kind'] == 'Withdrawal':
+        tx = pdict['mark']
```

Comment:
My idea with the `'mark'` field was actually to allow marking (flagging/labeling) the trade in some way in future. Like for example mark it as a payment, or just any special thing (IIRC, I think I used it in Poloniex import to mark trades as initiated from the maker as 'buy' or 'sell' or something). So I'd rather keep it reserved for future use.

I used the `'comment'` field for transaction IDs. But I see you actually used comment already in Electrum and Trezor wallet. Are those real comments? Should we rather introduce a new field, 'TransID' or something, then? (Sorry I didn't mention it earlier when the tplocs for these wallets were introduced - I didn't notice it at that time)

Reply:
I think a txid field would be a useful addition. Not all the exchanges I import from provide those in their exports, but it would be nice to have a record of them in the resultant data. Additionally, if txids are present, it could be another way (in the future) to help identify the correct deposit following a withdrawal.

And thanks for clarifying what the 'mark' column was intended for. I had been wondering 😄

Reply:
> Are those real comments?
Yes, in the wallet you can write some comments to each transaction and the exported `csv` files contain them.